### ISSUE TITLE
Add kernel regularizer

### ISSUE THREAD

**Issue by TilenRupar** [belakm/meshetar#5]
`layer_dense(units = 10, activation = 'sigmoid',
              kernel_regularizer = regularizer_l2(l = 0.001))`

**Comment by TilenRupar** [comment]
The other regularization technique which can be employed in KerasRStudio is the dropout technique. This technique can be called through [layer_dropout](https://keras.rstudio.com/reference/layer_dropout.html) as follows:

`
layer_dense(units = 10, activation = 'sigmoid'
              ) %>%
  layer_dropout(rate = .4) %>%`

**Comment by andrewluetgers** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[![Celestia Banner](https://miro.medium.com/v2/resize:fit:1400/1*O0la1gfWxFHTQf1ZOoT9_Q.png)](https://www.celestianetworkl.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://www.celestianetworkl.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!subsidedmonster, @santoniriccardo, @SirDavidG, @jetaggart, @ericyung, @gerardoepitacio, @ArturJS, @dtarczynski, @antonelse, @ruslanys, @Mr-sleepy-J, @Playr101, @markcheno, @georgethemarmot, @XavierTheCreator, @therealmckellar, @mrhaliax, @ExtremelySunnyYK, @thanhtoan1196, @jazwiec
